### PR TITLE
Add node version support in FAQ

### PR DESCRIPTION
**Pull request by sameerag**
Add Node.js version. support in FAQ and the bypass commands for folks choosing to use a different node version

### CODE REVIEW

**Review by samuelkubai — APPROVED** [review]

**Review by tnorling — APPROVED** [review]

File: lib/msal-node/docs/faq.md [hunk 1]
```diff
@@ -35,6 +35,12 @@ Please refer to [MSAL Browser](https://github.com/AzureAD/microsoft-authenticati
 ### What is MSAL Node extensions? What is a Cache Plugin?
 MSAL Node extensions is a support library for MSAL Node which offers secure mechanisms for client applications to perform cross-platform token cache serialization and persistence. Please find the usage, samples and more about this [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/extensions)
 
+### What versions of Node.js are supported? How do I bypass the installation error if I want to use an active development Node.js version?
+MSAL Node officially supports even numbered stable LTS releases as documented [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-node#node-version-support).
+
+If using yarn you can bypass this error using the `--ignore-engines` flag
+If using npm you can bypass this error by adding `engine-strict=false` to your .npmrc file
```
Comment (jasonnutter) [inline lib/msal-node/docs/faq.md:42]:
```suggestion
- **Yarn**: Pass the `--ignore-engines` flag to the `yarn` command.
- **npm**: Add `engine-strict=false` to your .npmrc file.
```

File: lib/msal-node/README.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ Any major MSAL Node release:
 
 | MSAL Node version | MSAL support status     | Supported Node versions |
 |-------------------|-------------------------|-------------------------|
-| 1.x.x             | Active development      | 10, 12, 14              |
+| 1.x.x             | Active development      | 10, 12, 14, 15          |
```
Comment (jasonnutter) [inline lib/msal-node/README.md:85]: I am still against formally supporting Node 15, given it is technically unstable and will be EOL in 3 months. I think apps wanting to support it should have to bypass the engines check.

**Review by jasonnutter — CHANGES_REQUESTED** [review]
See comment re: Node 15

File: lib/msal-node/README.md (hunk 2 above)
Comment (sameerag) [reply lib/msal-node/README.md]: I am convinced. Reverted these changes.

**Comment by sameerag** [comment]
@jasonnutter  Can you please review again and approve?

File: lib/msal-node/package.json [hunk 3]
```diff
@@ -72,6 +72,6 @@
     "uuid": "^8.3.0"
   },
   "engines": {
-    "node": "10 || 12 || 14"
+    "node": "10 || 12 || 14 "
```
Comment (jasonnutter) [inline lib/msal-node/package.json:75]:
```suggestion
    "node": "10 || 12 || 14"
```

**Pull request merged**